cli/cft.go

```diff
@@ -0,0 +1,12 @@
+package main
```

Comment:
Should this file be `main.go` by convention?

Reply:
it's always followed by conversion, for example no much main.go files in kubernetes repos

changed to main.go for more clarify